Tools/Scripts/webkitpy/layout_tests/run_webkit_tests.py

```diff
@@ -357,7 +357,7 @@ def parse_args(args):
             help=("Enable all GPU process related features, also set additional expectations and the result report flavor.")),
         optparse.make_option(
             "--site-isolation", action="store_true", default=False,
-            help=("Run each test in a cross origin iframe with and without site isolation enabled and compare the results. Uses site-isolation test expectations")),
+            help=("Run each test with site isolation enabled. Uses site-isolation test expectations")),
```

Comment:
As discussed on Slack, let's change it to "Run each test with and without site isolation enabled and compare the results."

Reply:
Do we have examples of folks using this type of workflow? While I agree that's a desirable feature, I'm not aware of folks using this type of workflow in `run-webkit-tests`. It feels out-of-scope for this change.